alternator/executor.cc

```diff
@@ -895,9 +917,12 @@ static void verify_billing_mode(const rjson::value& request) {
         if (!rjson::find(request, "ProvisionedThroughput")) {
             throw api_error::validation("When BillingMode=PROVISIONED, ProvisionedThroughput must be specified.");
         }
+        const rjson::value& throughput = request["ProvisionedThroughput"];
+        return billing_mode_type{true, throughput["ReadCapacityUnits"].GetInt(), throughput["WriteCapacityUnits"].GetInt()};
```

Comment:
GetInt will throw an ugly error is it's not an integer instead of an understandable error. You can use the existing function get_int_attribute() to do this for you, although this uses "int" and not the "uint64_t" you wanted to use. But you can also open-code the same thing.

Reply:
Changed and throw an exception if one of the values is missing